Feature model

magley opened the pull request:
Closes #78 

The class diagram is in `docs/`.
The main README has screenshots.

magley committed:
docs: Add class diagram

magley committed:
docs: Update class diagram

magley committed:
docs: Add screenshot to the README

Vasilijez reviewed (approved):
Not all functionalities are depicted in the screenshots, but I guess it would be overkill. It looks fine to me.

The class diagram looks nice.

hristinaina reviewed (approved):
Nice work, everything seems fine! I've noticed that you have included foreign keys as fields only in OrganizationMembers class and not in other classes.
I like that you added app screenshots!

magley commented:
> Not all functionalities are depicted in the screenshots, but I guess it would be overkill. It looks fine to me.

You're right, I tried to stick to things which were visually interesting. I had ~10 more screenshots that I removed because they weren't "impressive".

> I've noticed that you have included foreign keys as fields only in OrganizationMembers class and not in other classes.

Good catch, fixed in: [c67221](https://github.com/magley/mocker-hub/pull/79/commits/c6722117146b11c1d1ba54dd049efbb3a804c13f)

magley committed:
docs: Remove foreign keys from `OrganizationMembers`
